src/Bio/PDB/BondRestoring.hs

```diff
@@ -26,6 +28,12 @@ import           Linear.V3            (V3(..))
 
 import           Control.Monad        (guard)
 
+data AtomPDBId = AtomPDBId { atomSerial :: Int, atomChainID :: Char }
```

Comment:
I guess, it's better to move it into `Type.hs` as it's used not only here but in `PDB.hs` as well.